lib/memo_wise.rb

```diff
@@ -114,6 +114,38 @@ def self.has_kwarg?(method) # rubocop:disable Naming/PredicateName
     end
   end
 
+  # @private
+  #
+  # Determine whether `method` takes a *block* arg.
+  #
+  # There is one type of block arg:
+  #
+  #   * *Block* -- ex: `def foo(&block)`
+  #
+  # @param method [Method, UnboundMethod]
+  #   Arguments of this method will be checked
+  #
+  # @return [Boolean]
+  #   Return `true` if `method` accepts a block argument
+  #
+  # @example
+  #   class Example
+  #     def position_args(a, b=1)
+  #     end
+  #
+  #     def block_args(&block)
+  #       block.call
+  #     end
+  #   end
+  #
+  #   MemoWise.has_block_arg?(Example.instance_method(:position_args)) #=> false
+  #
+  #   MemoWise.has_block_arg?(Example.instance_method(:block_args)) #=> true
+  #
```

Comment:
These docs are maybe overkill, just copying what we had for `MemoWise.has_args?` and `MemoWise.has_kwargs?``